py/formatfloat.c

```diff
@@ -167,7 +144,6 @@ int mp_format_float(FPTYPE f, char *buf, size_t buf_size, char fmt, int prec, ch
             return s - buf;
         }
     }
-
```

Comment:
please don't add/remove unrelated blank lines unless it makes sense to the flow of the code that's being changed

Reply:
fixed